tutorenrollmentreports/templates/enrollmentreports/build/enrollmentreports/enrollment-report.yml

```diff
@@ -0,0 +1,8 @@
+# Emails enrollment reports.
+---
+- hosts: localhost
+  become: no
+  vars:
```

Comment:
Remove this `vars` dictionary please. We set `enrollment_report_mail_enable` in the Ansible inventory now.

Reply:
This is the final change that's still pending before we merge this, I believe. Please fix this, and then force-push it one more time.

Reply:
Done.